cmake/EthCompilerSettings.cmake

```diff
@@ -98,7 +98,12 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 			# Abort if linking results in any undefined symbols
 			# Note: this is on by default in the CMake Emscripten module which we aren't using
 			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -s ERROR_ON_UNDEFINED_SYMBOLS=1")
-			add_definitions(-DETH_EMSCRIPTEN=1)
```

Comment:
This one was unused, in case anybody is wondering why it's removed here.

Reply:
Was the "toolchain" PR merged? That should have removed it.

Reply:
Yep, that one was merged.